jupyter_dash/jupyter_app.py

```diff
@@ -17,10 +17,32 @@
 from ansi2html import Ansi2HTMLConverter
 import uuid
 
+from .comms import _dash_comm, _jupyter_config, _request_jupyter_config
 
-from werkzeug.debug.tbtools import get_current_traceback
 
-from .comms import _dash_comm, _jupyter_config, _request_jupyter_config
+def _get_skip(error: Exception, divider=2):
```

Comment:
Divider can be removed if refactor to raw traceback as it will be 1.

Reply:
If I understand correctly, the `divider` is always 1 for raw traceback, as it is [here](https://github.com/plotly/dash/blob/5e89d95225d17e4ae4b0fdd94fd1c05d7bfe8761/dash/dash.py#L139). I've updated the code accordingly.